### DISCUSSION TITLE
Sources (of dll, metadata) for code and doc located in different repos

### DISCUSSION THREAD

**Discussion by mikasoukhov** [dotnet/docfx#7583] [Q&A]
How to specify both of them and let docfx generate the right View Source and Improve doc at the same time with the right locations?

**Comment by JeremyCaney** [comment]
How are you sourcing your metadata for these repositories? It sounds like you have that working, but the “View Source” link isn’t populating. Can you share your `docfx.json`‘s `metadata` configuration for reference? I ask because I imagine the answer will depend on how you’re retrieving your metadata.
